### ISSUE TITLE
Update license to explictly allow distribution

### ISSUE THREAD

**Issue by vilhelmgray** [nashmuhandes/WidePix#2]
I'm including WidePix as part of a package for Gentoo and discovered that the WidePix license doesn't explicitly allow distribution of WidePix. This causes a problem for Gentoo because we prefer to mirror packages and assets on our own servers. Would you be able to update the WidePix license to explicitly allow distribution?

This should be as simple as modifying the license file line to:
```
You MAY use, copy, modify, merge, publish, distribute, and/or sublicense this work - HOWEVER:
```

That should make the meaning clearer that users are permitted to distribute WidePix and create derivative works, as opposed to just being allowed to make derivative works.

**Comment by madame-rachelle** [comment]
I am not 100% sure what Doom's license model allows here, this seems like it should be okay, but, Doom's license is not exactly "free-for-all" when it comes to art derived from the game. It is Bethesda/id Software who would have tied our hands here, not Nash.

Part of me wants to say this should be a problem-free change and could easily be done - but another part of me is very unsure about it.

**Comment by madame-rachelle** [comment]
I think I should also voice the usual caveat of "I am not a lawyer" - I think in this instance, we should get one. Or at least, it would be nice to have Bethesda's legal department comment on this.

**Comment by chewi** [comment]
There's a tiny chance I might be able to pull some strings here but probably not until Microsoft has finished the acquisition in a few months.

**Comment by vilhelmgray** [comment]
IANAL, but I think we determined last year that derivatives of Doom assets were allowed to be created and distributed: https://forum.zdoom.org/viewtopic.php?f=124&t=65291&sid=00505eb95e5922b33e7620f597eb97c9

This is because Section 2 of the [Doom Collector's Edition EULA](https://gitweb.gentoo.org/repo/gentoo.git/plain/licenses/DOOM-COLLECTORS-EDITION) explicitly permits creation and distribution of derivative works of art:
```
2. Permitted New Creations.  Subject to the terms and provisions of this Agreement and so long as you fully comply at all times with this Agreement, ID grants to you the non-exclusive and limited right to create for the Software (except any Software code) your own modifications (the "New Creations") which shall operate only with the Software (but not any demo, test or other version of the Software).
...
You are only permitted to distribute, without any cost or charge, the New Creations to other end-users so long as such distribution is not infringing against any third party right and is not otherwise illegal or unlawful.
```

The only problem the Gentoo team is having is that the WidePix license inadvertently adds an additional limitation restricting distribution because it only states the creation of derivate works is permitted.

**Comment by madame-rachelle** [comment]
I'll ask Nash if it's okay to make the change - but please bear in mind I am only looking out for you with expressing my concerns here. If you are confident in Bethesda's stance on this, then I doubt there will be any issue with the actual change as requested.

**Comment by vilhelmgray** [comment]
@chewi Do you have a relationship with the Microsoft acquisition that could change the licensing of the Doom assets?

**Comment by vilhelmgray** [comment]
Thanks @madame-rachelle I really appreciate all the help! :+1:

Regarding Bethesda, I'm certain they'd end up giving the same boilerplate copyright response as when I asked about use of the Wolfenstein assets earlier this year: https://bugs.gentoo.org/703610

Fortunate for us however, we have the Doom EULA giving explicit permission for the distribution and derivate works cases; otherwise no modded Doom assets by anyone would be permitted and WidePix wouldn't be allowed to exist at all. I think the WidePix license covers compatibility with the Doom EULA explicitly by enumerating it in the list so we should be safe:
```
Your work must comply with id Software's original licenses (i.e. they must only work on the game from which they originated)
```

**Comment by chewi** [comment]
Well, I now work for a completely different part of Microsoft but there are central contacts for licensing and I received some licensing training just the other day.

This actually doesn't stop Gentoo from using WidePix at all as users can still fetch directly from here if we set the right flags. It's really more of a problem for other distributions.

**Comment by vilhelmgray** [comment]
> This actually doesn't stop Gentoo from using WidePix at all as users can still fetch directly from here if we set the right flags. It's really more of a problem for other distributions.

That's a fair point, Gentoo packaging is flexible enough that we could setup the Gentoo gzdoom package to simply fetch WidePix directly from Github instead of mirroring it. I suspect the other distributions will soon be requesting this license update too however (because they prebuild their binary packages), but it could give you some time to get more info on the Doom licensing situation if you want to go down that route.

**Comment by madame-rachelle** [comment]
Done. https://github.com/nashmuhandes/WidePix/commit/92738042ca3a37f28153a09809d80a7d61090532
